scripts/export_data.sh

```
@@ -0,0 +1,35 @@
+EXPORT_FORMAT=${1:-json}
+OUTPUT_DIR=${2:-/tmp/osidb-exports}
+API_KEY=${3}
+
+mkdir -p $OUTPUT_DIR
+
+TMPFILE=$(mktemp)
+trap "rm -f $TMPFILE" EXIT
+
+echo "Starting export with API key: $API_KEY"
+
+QUERY="SELECT * FROM flaws WHERE status = 'active'"
```

Shellcheck (0.11.0)

[warning] QUERY appears unused. Verify use (or export if used externally).

(SC2034)